frontend/public/components/start-guide.jsx

```diff
@@ -54,12 +56,14 @@ export const StartGuidePage = () => <div className="co-p-has-sidebar">
   <DocumentationSidebar />
 </div>;
 
-export const OpenShiftGettingStarted = () => <div className="co-well">
+const OpenShiftGettingStarted_ = ({createProjectMessage}) => <div className="co-well">
   <h4>Getting Started</h4>
   <p>
-    OpenShift helps you quickly develop, host, and scale applications. Create a project for your application.
+    OpenShift helps you quickly develop, host, and scale applications. {createProjectMessage || 'To get started, create a project for your application.'}
```

Comment:
I might remove the generic description here when there's a custom message to show.

```jsx
{createProjectMessage ?
  <p>{createProjectMessage}</p>
  <p>OpenShift helps you quickly develop, host, and scale applications. Create a project for your application.</p>}
```

Reply:
Did you mean something like this?
```jsx
  { createProjectMessage
    ? <p className="co-pre-line">{createProjectMessage}</p>
    : <p>
        OpenShift helps you quickly develop, host, and scale applications.
        To get started, create a project for your application.
    </p>
  }
```

Reply:
Yeah exactly